api/prometheus/v1/api.go

```diff
@@ -404,6 +407,20 @@ type queryResult struct {
 	v model.Value
 }
 
+// TSDBResult contains the result from querying the tsdb endpoint
+type TSDBResult struct {
```

Comment:
I am maybe bias, but I wish we could just import https://github.com/prometheus/prometheus/blob/master/web/api/v1/api.go#L1172

This is fine though I guess - to avoid importing prometheus.

Reply:
It might to avoid cyclic dependencies? In any case, I would stick to what already is standard practice here but we could open issue for this to discuss?